transforms/setup-config-with-workspaces/__testfixtures__/basic.input.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function (defaults) {
+  let app = new EmberApp(defaults, {
+    'ember-cli-pemberly-i18n': i18nConfig,
```

Comment:
Can we genericize these fixtures so that we're not using internal LI specific tooling?